pkg/virt-controller/watch/vmi.go

```diff
@@ -1056,6 +1095,18 @@ func (c *VMIController) sync(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod,
 	}
 
 	if !isTempPod(pod) && isPodReady(pod) {
+		if vmispec.SRIOVInterfaceExist(vmi.Spec.Domain.Devices.Interfaces) {
```

Comment:
Is there a special reason there are no unit tests?

Reply:
I forgot to add it here, but now I am hitting some issues..

Reply:
Done